components/finance/SalesTaxQuestion.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { useState, useEffect } from "react";
+import { GuessData } from "../../pages/api/guessData";
+import { name } from "../../pages/api/names";
+import { Button } from "../ui/Button";
+
+export interface SalesTaxQuestionProps {
+  personName: string;
+  number: number;
+  taxRate: number;
+  price: number;
+  submitGuess: (guess: GuessData) => void;
+  multipleAnimals: string;
+  image2: string;
```

Comment:
why is this called `image2` instead of `image`?

Reply:
no good reason. I changed it.  It makes sense why it needs to be changed.